vispy/visuals/visual.py

```diff
@@ -649,7 +649,7 @@ def _compute_bounds(self, axis, view):
                 vb = v.bounds(axis)
                 if bounds is None:
                     bounds = vb
-                else:
+                elif vb is not None:
```

Comment:
This was a bug that I caught: the `CompoundVisual` would access the bounds of the sub-visual, but wouldn't check if the sub-visual returned `None` as its bounds.